bigquery/google/cloud/bigquery/client.py

```diff
@@ -1439,11 +1439,12 @@ def _check_mode(stream):
     :param stream: A bytes IO object open for reading.
 
     :raises: :exc:`ValueError` if the ``stream.mode`` is a valid attribute
-             and is not among ``rb``, ``r+b`` or ``rb+``.
+             and is not among ``rb``, ``r+b``, ``rb+``, or, in case of gzip,
+             not among ``1`` or ``2``.
```

Comment:
Can you please use the named constant, `gzip.READ`, rather than `1`?  Also, I don't believe `2` (aka `gzip.WRITE`) should be an allowable value:  we explicitly want a readable stream.